.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+# This is a basic workflow to help you get started with Actions
```

Comment:
Maybe this comment should be updated. ;-)

Reply:
It's a template.  Not too worried :-)